telos-frontend/src/components/calendar/DashboardCalendar.jsx

```diff
@@ -1,36 +1,159 @@
 import { Calendar, momentLocalizer } from 'react-big-calendar';
+import { useState } from 'react';
+import TextField from '@material-ui/core/TextField';
 import moment from 'moment';
+import Button from '@material-ui/core/Button';
```

Comment:
Would it be neater to import like:
import { Button, Dialog, etc } from '@material-ui/core'?

Reply:
yeah, good point. I'll do that. Thanks!